pkg/api/validation/validation.go

```diff
@@ -839,9 +839,6 @@ func validateContainers(containers []api.Container, volumes util.StringSet) errs
 		} else {
 			allNames.Insert(ctr.Name)
 		}
-		if len(ctr.Image) == 0 {
```

Comment:
Please leave the validation in place for now. This is going to be one field of a "OneOf", but currently is the only field.